scripts/build.sh

```diff
@@ -91,7 +93,13 @@ build() {
     GIT_COMMIT="$(git describe --match=NeVeRmAtCh --always --abbrev=7 --dirty)"
 
     echo "** Building & installing dfuseeos binary for $GIT_COMMIT **"
-    go install -ldflags "-X main.commit=$GIT_COMMIT" ./cmd/dfuseeos
+    # ultra-duncan --- disable symbol table by default
+    if [[ $enable_symbol_table != true ]]; then
+      echo "** Enable symbol table only work with go lang version is < 1.20. **"
```

Comment:
I'm using go lang 1.20.x, the build only works if I build with -t, i.e. enable the symbol table.